web/module/machines.py

```diff
@@ -45,7 +45,7 @@ def handle_exceptions(request, exception):
     check_payload_deploy(request)
 
     with data.Connection.use() as conn:
-        new_request = data.Request(state='created', type='deploy')
+        new_request = data.Request(type='deploy')
```

Comment:
removed explicit state setting for `RequestState.CREATED`

Reply:
I know it is explicit but even when it is explicit it may be more readable; what do you think?

Reply:
on the other way it is assigned to the new_request var. it is obvious; I approved the request